src/Kestrel.Transport.Libuv/Internal/ListenerContext.cs

```diff
@@ -38,6 +41,43 @@ protected UvStreamHandle CreateAcceptSocket()
             }
         }
 
+        protected async Task HandleConnectionAsync(UvStreamHandle socket)
```

Comment:
I'm not a fan of this being in the patch. Is this what's currently checked into dev?

Reply:
This is what's in dev. Even though we don't need to dispose CancellationTokenSources, I think it important for anything implementing connection features including the transport to know when the app/middleware is done executing and no longer needs access to the connection features it implements.

Just relying on Pipe APIs to determine when to cleanup resources backing connection features is insufficient. The HttpMiddleware, for example, still should be able access connection features after HttpContext.Abort() is called.

All that said, having a Task returning IConnectionDispatcher.OnConnection isn't critical to this backport given that it's acceptable to never dispose ConnectionClosed CancellationTokenSources, so I reverted that aspect of this backport.